test/test-styles.js

```diff
@@ -0,0 +1,79 @@
+import { assert, it } from './x-test.js';
+import styleSheet from './test-styles.css.js';
+import XElement from '../x-element.js';
+
+class TestElement1 extends XElement {
+  static count = 0;
+  static get styles() {
+    TestElement1.count++;
+    return [styleSheet];
+  }
+  static template(html) {
+    return () => {
+      return html``;
+    };
+  }
+}
+customElements.define('test-element-1', TestElement1);
+
+it('provided style sheets are adopted', () => {
+  const el = document.createElement('test-element-1');
+  document.body.append(el);
+  const boundingClientRect = el.getBoundingClientRect();
+  assert(boundingClientRect.width === 100);
+  assert(boundingClientRect.height === 100);
+  el.remove();
+});
+
+it('should only get styles _once_ per constructor', () => {
```

Comment:
Importantly, this function is called _once_ since it’s a static member, we follow our typical pattern — assume it can be set up once during static analysis and re-used for all instances.